packages/expo-sms/ios/ExpoSMS.podspec

```diff
@@ -3,7 +3,7 @@ require 'json'
 package = JSON.parse(File.read(File.join(__dir__, '..', 'package.json')))
 
 Pod::Spec.new do |s|
-  s.name           = 'EXSMS'
+  s.name           = 'ExpoSMS'
   s.version        = package['version']
   s.summary        = package['description']
   s.description    = package['description']
```

Comment:
AFAIK all the other modules also specify the used Swift version here. Maybe something that should be added?